codegen/codegen.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	defaultURL = "https://releases.rancher.com/kontainer-driver-metadata/dev-v2.8/data.json"
+	defaultURL = "https://releases.rancher.com/kontainer-driver-metadata/release-v2.8/data.json"
```

Comment:
Same as the other PR, there is no need to change the url to the release branch, unless you plan to have a following PR to change it back to dev (as a part of the post-release cleaning up).